packages/grafana-ui/src/components/Slider/styles.ts

```diff
@@ -33,28 +38,31 @@ export const getStyles = stylesFactory((theme: GrafanaTheme, isHorizontal: boole
         margin-top: -10px;
       }
       .rc-slider-handle {
-        border: solid 2px ${theme.palette.blue77};
-        background-color: ${theme.palette.blue77};
+        border: none;
+        background-color: ${blueColor};
+        cursor: pointer;
       }
       .rc-slider-handle:hover {
-        border-color: ${theme.palette.blue77};
+        box-shadow: 0px 0px 0px 6px ${blueOpacity};
```

Comment:
Should this be abstracted somewhere as it used many times?

Reply:
yes, should be moved into a variable